src/main/java/seedu/agendum/logic/commands/StoreCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.agendum.logic.commands;
+
+import seedu.agendum.commons.core.Config;
+import seedu.agendum.commons.util.FileUtil;
+import seedu.agendum.commons.util.StringUtil;
+
+/**
+ * Allow the user to specify a folder as the data storage location
+ */
+public class StoreCommand extends Command {
+    
+    public static final String COMMAND_WORD = "store";
+    public static final String MESSAGE_SUCCESS = "New save location: %1$s";
+    public static final String MESSAGE_LOCATION_INVALID = "The specified location is invalid.";
+    public static final String MESSAGE_LOCATION_DEFAULT = "Save location set to default: %1$s";
+    
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Specify a save location. \n"
+            + "Parameters: FILE_PATH\n" 
+            + "Example: " + COMMAND_WORD 
+            + "C:/agendum";
+    private String newSaveLocation;
+
+    public StoreCommand(String location) {
+        newSaveLocation = location.trim();
+    }
+
+    @Override
+    public CommandResult execute() {
+        assert newSaveLocation != null;
+        
+        if(newSaveLocation.equalsIgnoreCase("default")) {
+            String defaultLocation = Config.DEFAULT_SAVE_LOCATION;
+            model.changeSaveLocation(defaultLocation);
+            return new CommandResult(String.format(MESSAGE_LOCATION_DEFAULT, defaultLocation));
+        }
+
+        if(!isNewSaveLocationValid()) {
+            indicateAttemptToExecuteIncorrectCommand();
+            return new CommandResult(MESSAGE_LOCATION_INVALID);
+        }
+
+        model.changeSaveLocation(newSaveLocation);
+        return new CommandResult(String.format(MESSAGE_SUCCESS, newSaveLocation));
+    }
+    
+    private boolean isNewSaveLocationValid() {
```

Comment:
I think it might be better to check/parse in Parser class prepareStore e.g. not a empty string, isStringValid
Currently, when an invalid file path is provided, the message (MESSAGE_LOCATION_INVALID) is not very informative.
It might be more helpful to show MESSAGE_USAGE.

Check if the file path is valid within this command. Follow coding standards: isLocationValid

- [ ] Fixed